app/src/main/java/com/example/easysurvey/MainActivity.kt

```diff
@@ -23,40 +28,78 @@ class MainActivity : ComponentActivity() {
             EasySurveyTheme {
                 // A surface container using the 'background' color from the theme
                 Surface(modifier = Modifier.fillMaxSize(), color = MaterialTheme.colors.background) {
-                    HomepageButtons()
+                    homepageButtons()
                 }
             }
         }
     }
 }
 
 @Composable
-fun HomepageButtons() {
+fun homepageButtons() {
     val text = "You clicked this button"
     val duration = Toast.LENGTH_LONG
     val context = LocalContext.current
-    Column{
-        Button(
-            onClick = { 
-                Toast.makeText(context, text, duration).show()
+    Column(
+        Modifier.fillMaxHeight(),
+        verticalArrangement = Arrangement.SpaceBetween)
+    {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(130.dp)
+                .padding(vertical = 30.dp, horizontal = 30.dp),
+            horizontalArrangement = Arrangement.Center,
+        ) {
+            Column(horizontalAlignment = Alignment.CenterHorizontally) {
+                Text(
+                    text = stringResource(id = R.string.app_name),
+                    style = TextStyle(fontSize = 40.sp)
+                )
+                Text(
+                    text = stringResource(id = R.string.app_description),
+                    style = TextStyle(fontSize = 13.sp)
+                )
             }
-        ){ Text(text = "Take a Survey")}
-        Button(
-            onClick = {
+        }
+        Row(
+            horizontalArrangement = Arrangement.SpaceBetween,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(8.dp)
+        ){
+            Button(onClick = {
                 Toast.makeText(context, text, duration).show()
+            },
+                shape = RoundedCornerShape(20.dp)
+            ) {
+                Image(
+                    painterResource(id = R.drawable.ic_baseline_create_24),
+                    contentDescription = "Pencil icon",
+                    modifier = Modifier.size(20.dp)
+                )
+                Text(text = "Take a Survey", Modifier.padding(start = 10.dp))
             }
-        ){ Text(text = "Create a Survey")}
-        Button(
-            onClick = {
+
+            Button(onClick = {
                 Toast.makeText(context, text, duration).show()
+            },
+                shape = RoundedCornerShape(20.dp)
+            ) {
+                Image(
+                    painterResource(id = R.drawable.ic_baseline_add_circle_24),
+                    contentDescription = "Add icon",
+                    modifier = Modifier.size(20.dp)
+                )
+                Text(text = "Create a Survey", Modifier.padding(start = 10.dp))
             }
-        ){ Text(text = "See Survey Results")}
+        }
     }
 }
```

Comment:
I changed how the buttons were laid out on the splash screen. Before this, they were floating at the top left corner of the screen. I changed it so that instead they sit at the bottom of the screen, evenly spaced from each other. I also opted to remove the "See Survey Results" button, as I think that the user should only see that after logging in.